bus-mapping/src/evm/opcodes/begin_end_tx.rs

```diff
@@ -11,6 +11,8 @@ use eth_types::{
 };
 use ethers_core::utils::get_contract_address;
 
+const PRECOMPILE_COUNT: u64 = 9;
```

Comment:
minor suggestion: move `PRECOMPILE_COUNT` declaration to eth-types/src/evm_types.rs then we don't need to define `PRECOMPILE_COUNT` in different files.

Reply:
Good point. I addressed it in [bb22082](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1640/commits/bb220825358c4a6203998245d7f7e0e302821b98)